lib/api/attributes.js

```diff
@@ -1,10 +1,12 @@
 var _ = require('lodash'),
+  static = require('../static'),
```

Comment:
I used this style for consistency with the file as is. I suggest that if we make that change then all other `utils` should have a similar format:

```js
   isTag = require('../utils').isTag,
   domEach = require('../utils').domEach,
   hasOwn = Object.prototype.hasOwnProperty,
   camelCase = require('../utils').camelCase,
   cssCase = require('../utils').cssCase,
```